<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BCC_B6000</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Transmission</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(215, 222, 223);</string>
          </property>
          <property name="title">
           <string>Recived</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTableWidget" name="ctl_tableRciv"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(215, 222, 223);</string>
          </property>
          <property name="title">
           <string>Send</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTableWidget" name="ctl_tableSend"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Config</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>50</y>
          <width>231</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(215, 222, 223);</string>
        </property>
        <property name="title">
         <string>BCC_B6000</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>IP</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ctl_IP_BCC">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>113</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="ctl_portBCC">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>70</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>50</y>
          <width>231</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(215, 222, 223);</string>
        </property>
        <property name="title">
         <string>FCC</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>IP</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ctl_IP_FCC">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>113</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="ctl_portFCC">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>70</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>50</y>
          <width>231</width>
          <height>121</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(215, 222, 223);</string>
        </property>
        <property name="title">
         <string>RFC</string>
        </property>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>IP</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>80</y>
           <width>59</width>
           <height>15</height>
          </rect>
         </property>
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="ctl_IP_RFC">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>113</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="ctl_portRFC">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>70</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1228</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
